test/integration_test.py

```diff
@@ -408,6 +409,22 @@ def __check_manifest(self, file: IO[bytes], uuid_field_name: str) -> List[Mappin
         self.assertEqual(bundle_uuid, str(uuid.UUID(bundle_uuid)))
         return rows
 
+    def _check_curl_manifest(self, _catalog: CatalogName, response: bytes):
+        text = TextIOWrapper(BytesIO(response))
+        urls, names = [], []
+        for line in text.readlines():
+            if line.startswith('--') or line == '':
+                pass  # skip empty lines and config lines like '--compressed\n'
+            elif m := re.fullmatch(r'url="(.+)"\n', line):
+                urls.append(m.group(1))
```

Comment:
This is a little more verbose, but consider
```suggestion
            elif m := re.fullmatch(r'url="(?P<url>.+)"\n', line):
                urls.append(m.group('url'))
```
in order to avoid the "magic" 1.

Reply:
Replaced regex check with a more simple string verification of the expected lines in the manifest.
Ticket #2622 will extend the integration test to run curl with the manifest and verify the results.